contracts/strategies/allocation/rfp-simple/README.md

```diff
@@ -2,72 +2,37 @@ Spec: Simple Request for Proposal (RFP)
-- `poolOpen`
-- `rfpMetadata`
-- `decisionMaker`
-
-#### `registerRecipients()`
-Potential applicants can apply to the pool via `registerRecipients`. When the recipient is submitted, the strategy uses the following decision tree to determine eligibility:
-
-- Is `poolOpen` true?
-    - If yes, proceed
-    - If no, revert with message that pool is not accepting proposals
-- Does the applicant have a valid Allo registry identity?
-    - If yes, proceed
-    - If no, revert that valid registry identity is required
-- Does `proposal` contain data? 
-    - If yes, proceed
-    - If no, revert with message that pool requires a proposal
-- Does `proposalBid` contain a value?
-    - If yes, the recipient is eligible
-    - If no, revert with message that specific bid must be requested
-
-If the recipient is eligible, the strategy stores the recipient and programmatically assigns the `Pending` local status.
-
-#### `allocate()`
-The decision maker is able to choose which proposal (if any) will win the RFP using the `allocate` function. When a new allocate transaction is submitted, the following checks are made:
-- Does msg.sender == `decisionMaker`?
-    - If yes, record vote
-    - If no, revert with message that voter is not eligible
-
-Within the `allocate` function, the decision maker is able to give one of two commands:
-- Select a winning recipient —> this marks the recipient as `accepted` and all other recipients as `rejected`.
-- Choose not to accept bids —> this marks all recipients as `rejected`
-
-#### `generatePayouts()`
-The `generatePayouts` function checks if there is an accepted recipient.
-- If there is an `accepted` recipient, that recipient is moved to the distribution process
-- If all recipients are `pending`, the transaction reverts with the message that a winner has not been selected yet
-- If all recipients are `rejected`, the funds are made available to be reclaimed by the pool admin
-
-### Open questions
-- do we need a global "ineligible" status?
\ No newline at end of file
+### Recipient logic
+In this strategy, prospective recipients only need to apply for a grant. There isn't an approval step before voting. 
+- `registerRecipient` - function for recipient to submit application
+    - if a recipient submits an application that meets all requirements, then the recipient status is set to `Pending`
+    - **Customizable Settings**
+        - pool admin can require recipients to have Allo registry identity in order to apply
+            - if pool admin doesn't require Allo identity and the recipient doesn't have one, then `msg.sender` is used as recipient id
+        - pool admin can include the full RFP, as well as optional and required questions in an application, stored in metadata
+            - pool admin must specify the maximum amount they are willing to pay — this is the pool amount stored on the contract. 
```

Comment:
why does this come into the recipient logic ? The upper limit -> if needed should be done on initializing the contract

Reply:
I've removed this